### ISSUE TITLE
WindowManager MessageBox UI issue

### ISSUE THREAD

**Issue by zsonglorman** [canton7/Stylet#138]
Dear,

First of all, thank you for all your work, I find this project truly useful for my developments.

I'm using the WindowManager to show Stylet's own MessageBox as discussed in the docs:

`this.windowManager.ShowMessageBox($"Error message here.", "Warning", MessageBoxButton.YesNoCancel, MessageBoxImage.Warning);`

It works, however, the buttons on the MessageBox are strange-looking:

![messagebox](https://user-images.githubusercontent.com/19626247/86510298-d3b5e200-bdee-11ea-8585-785eb30f3203.PNG)

Do you have any idea what could cause this issue?
Thanks a lot in advance!

**Comment by canton7** [comment]
That's very weird, I haven't seen that before. If you create a `<Button>` yourself, does it look the same?

**Comment by zsonglorman** [comment]
Yes, it's very strange, and all other controls including buttons work just fine in windows, usercontrols etc.

**Comment by kaisame** [comment]
try change the font size

**Comment by Coke21** [comment]
Hi, sorry for the thread bumping but I'm also having the same problem:
`_windowManager.ShowMessageBox("test", "Test", MessageBoxButton.OK, MessageBoxImage.Hand);`
Shows:
![](https://i.gyazo.com/fb185476a646de18fdb8f1e87f393e18.png)

I'm not sure if it helps but I'm using ModernWpf as UI framework -> https://github.com/Kinnara/ModernWpf
Does anyone know how to fix the button? Thanks

**Comment by mortan** [comment]
Hi there, I got the same problem as @Coke21. My messagebox button is a) not centered and b) clipping.

![grafik](https://user-images.githubusercontent.com/1070913/121402107-cc317e00-c959-11eb-8d32-ee3d0c23626c.png)

I would imagine that the "not centered" part is coming from this code (unfortunately I can't test it):

![grafik](https://user-images.githubusercontent.com/1070913/121402175-dd7a8a80-c959-11eb-97a5-44fdd9695e90.png)

Would be awesome if this gets fixed, if you need anything let me know.

**Comment by steveyan00** [comment]
I'm using [HandyControl](https://github.com/HandyOrg/HandyControl) as UI framework and I'm having the same problem. Also, MessageBox cannot adapt to dark mode.
Is it possible to allow customizing the style of MessageBox in future versions?
